api/src/main.rs

```diff
@@ -327,16 +323,6 @@ async fn start_main_server(config: &'static Config, authorization: Authorization
                     .layer(RequestDecompressionLayer::new()),
             ),
         )
-        .route(
-            "/smith/telemetry/:service",
-            post(telemetry::routes::service).layer(
```

Comment:
can we change this to just return OK, at least until we are sure all services are not using it anymore